.github/ISSUE_TEMPLATE/bug_report_template.yaml

```diff
@@ -101,40 +109,76 @@ body:
         with mlflow.start_run():
             mlflow.sklearn.log_model(model, "model")
         ```
+
+      value: |
+        <!-- PLEASE KEEP BACKTICKS AND CHECK PREVIEW -->
+        ```
```

Comment:
```suggestion
        ``` python
```

Reply:
The color coding helps read walls of script :)

Reply:
The code could be R or Java :)